packages/mui-lab/src/MonthPicker/PickersMonth.tsx

```diff
@@ -27,21 +27,28 @@ const PickersMonthRoot = styled<
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
-  cursor: 'pointer',
-  height: 64,
+  color: 'unset',
+  backgroundColor: 'transparent',
+  border: 0,
   outline: 0,
-  transition: theme.transitions.create('font-size', { duration: '100ms' }),
-  '&:focus': {
-    color: theme.palette.primary.main,
-    fontWeight: theme.typography.fontWeightMedium,
+  ...theme.typography.subtitle1,
+  margin: '8px 0',
+  height: 36,
+  borderRadius: 16,
```

Comment:
Small detail: I believe the container should be fully rounded, as shown in [Material's documentation](https://material.io/components/date-pickers#mobile-pickers)! How's the best way to do it? Just throwing a 99 or something?

<img width="300" alt="Screen Shot 2021-11-11 at 12 15 38" src="https://user-images.githubusercontent.com/67129314/141322604-53b44e86-9bea-43e0-a23b-1e2c0583c5a7.png">

Reply:
Awesome, thank you!